PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# My Awesome Pull Request
+
+## What does this change?
+
+## What does success look like and how do you plan to measure success?
```

Comment:
This seems very specific to certain, feature / test based PRs.
What about something more generic, e.g. "What's the value of this?" making sure we're not doing things for the sake of it?